packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -106,7 +106,20 @@ export default defineConfig({
   }
 })
 
-const ethRpcUrl = process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL
+const ethRpcUrl = (() => {
+  // MetaMask comes with a default http://localhost:8545 network with 'localhost' as network name
+  // On CI, the rpc is http://geth:8545 so we cannot reuse the 'localhost' network
+  // However, Synpress does not allow editing network name on the MetaMask extension
+  // For consistency purpose, we would be using 'custom-localhost'
+  // MetaMask auto-detects same rpc url and blocks adding new custom network with same rpc
+  // so we have to add a / to the end of the rpc url
+  if (
+    process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL === 'http://localhost:8545'
```

Comment:
In case the URL changed in the future, it'd be safer to do something like:

```suggestion
    (process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL.endsWith('/') {
       return process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL
    }
    return process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL + '/'
```

Reply:
was using this locally earlier but decided to use localhost
will revert